cea/optimization/individual_evaluation/individual_evaluation.py

```diff
@@ -329,21 +328,21 @@ def individual_evaluation(individual, building_names, total_demand, locator, ext
     CO2 += addCO2 + coolCO2
     prim += addPrim + coolPrim
     # Converting costs into float64 to avoid longer values
-    costs = np.float64(costs)
-    CO2 = np.float64(CO2)
-    prim = np.float64(prim)
+    costs = np.float64(costs).round(2)
+    CO2 = np.float64(CO2).round(2)
+    prim = np.float64(prim).round(2)
 
     # add electricity costs corresponding to
 
-    print ('Additional costs = ' + str(addCosts))
-    print ('Additional CO2 = ' + str(addCO2))
-    print ('Additional prim = ' + str(addPrim))
+    # print ('Additional costs = ' + str(addCosts))
+    # print ('Additional CO2 = ' + str(addCO2))
+    # print ('Additional prim = ' + str(addPrim))
 
-    print ('Total costs = ' + str(costs))
-    print ('Total CO2 = ' + str(CO2))
-    print ('Total prim = ' + str(prim))
+    print ('Total costs [$/yr] = ' + str(costs))
```

Comment:
great about the units. However bear in mind that most of the time they are reported in tons and MJ/year.

Reply:
changed to ton/year